src/main/java/org/repodriller/persistence/csv/CSVFile.java

```diff
@@ -18,14 +15,18 @@
 
 import java.io.FileOutputStream;
 import java.io.PrintStream;
-import org.apache.commons.lang3.StringEscapeUtils;
+import java.util.Arrays;
 
+import org.apache.commons.lang3.ArrayUtils;
+import org.apache.commons.lang3.StringEscapeUtils;
 import org.repodriller.persistence.PersistenceMechanism;
 
 public class CSVFile implements PersistenceMechanism {
 
 	private PrintStream ps;
-
+	private String[] header = new String[0];
```

Comment:
What happens if the user doesn't pass a header? Are you sure it works? Maybe an automated test to proof it? :)